app/src/App.vue

```diff
@@ -102,6 +102,30 @@ setTimeout(() => {
         { immediate: true },
     );
 }, 1000);
+
+setTimeout(() => {
+    watch(
+        [isConnected, isAuthenticated],
+        () => {
+            if (isConnected.value && !isAuthenticated.value) {
+                useNotificationStore().addNotification({
+                    id: "accountBanner",
+                    title: "You are missing out !",
```

Comment:
Nitpick: remove the space before the exclamation mark.

```suggestion
                    title: "You are missing out!",
```